csrc/ops/alias.cpp

```diff
@@ -744,7 +747,8 @@ TensorView* slice(TensorView* inp, const std::vector<Slice>& ranges) {
       ", Expected: ",
       ndims);
 
-  const auto normalize_slice_range = [](Slice range, Val* extent) -> Slice {
+  const auto normalize_slice_range =
+      [](Slice range, Val* extent, bool manual_normalization) -> Slice {
```

Comment:
Can you just capture `manual_normalization` instead of passing it explicitly?

Reply:
Sure. I tend to capture when it become inconvenient to pass something as an argument.